### ISSUE TITLE
collection-repeat not work in android under 4.4

### ISSUE THREAD

**Issue by afelipe** [driftyco/ionic-bower#15]
collection repeat works perfectly in IOS and android devices 4.4 (and higher) but when I do scroll  in android 4.3 (and lower) the list disappear.

**Comment by toddhalfpenny** [comment]
I am also seeing this in Android 4.3.
Code that seems to be working fine in 4.4 doesn't show the list in 4.3.
If I change the _collection-repeat_ to an _ng-repeat_ then it shows OK. I notice this using release  v1.0.0-beta.14-nightly-1102

**Comment by toddhalfpenny** [comment]
Looking into the logcat I can see this;
```D/CordovaLog(20153): /lib/js/ionic.bundle.min.js: Line 19516 : TypeError: Object #<Text> has no method 'contains'
D/CordovaLog(20153):     at /lib/js/ionic.bundle.min.js:49427:18
D/CordovaLog(20153):     at NodeList.filter (native)
D/CordovaLog(20153):     at postLink (/lib/js/ionic.bundle.min.js:49426:34)
D/CordovaLog(20153):     at invokeLinkFn (/lib/js/ionic.bundle.min.js:16128:9)
D/CordovaLog(20153):     at nodeLinkFn (/lib/js/ionic.bundle.min.js:15638:11)
D/CordovaLog(20153):     at compositeLinkFn (/lib/js/ionic.bundle.min.js:14987:13)
D/CordovaLog(20153):     at compositeLinkFn (/lib/js/ionic.bundle.min.js:14990:13)
D/CordovaLog(20153):     at nodeLinkFn (/lib/js/ionic.bundle.min.js:15633:24)
D/CordovaLog(20153):     at compositeLinkFn (/lib/js/ionic.bundle.min.js:14987:13)
D/CordovaLog(20153):     at compositeLinkFn (/lib/js/ionic.bundle.min.js:14990:13)
```
Looks like it's related to the recent addition of the 4 collection-repeat modes.

**Comment by ajoslin** [comment]
Fixed via 6c5555469c3c0f38868eab898b304600681d53a5

In the future, please open issues on the main ionic repo.
